Tree/readme.md

```diff
@@ -47,6 +47,7 @@ that, this property should be satisfied at every node in the tree.
 * Sorting the elements of binary search tree and many more
 
 ## Questions :
```

Comment:
again sort the list according to name as earlier.

Reply:
done sir, please review